src/plugins/reporter-rhtsupport.c

```diff
@@ -644,7 +648,10 @@ int main(int argc, char **argv)
                 int yes = ask_yes_no(msg);
                 free(msg);
                 if (!yes)
+                {
+                    g_free(url);
```

Comment:
Freed `url` is used on the following line.

Reply:
Ugh, every time.